tests/integration/cocoon/test_cocoon_instruments.py

```diff
@@ -782,3 +782,81 @@ def test_load_from_data_frame_instruments_with_strip(
                 for key, value in expected_outcome["instruments"].values.items()
             )
         )
+
+    def test_load_instrument_properties(
+        self,
+    ):
+        data_frame = pd.DataFrame({
+            "instrument_name": ["BP_LondonStockEx_BP", "Glencore_LondonStockEx_GLEN", "TESCO_LondonStockEx_TSCO"],
+            "figi": ["BBG000C05BD1", "BBG001MM1KV4", "BBG000BF46Y8"],
+            "isin": ["BBG000C05BD1", "JE00B4T3BW64", "GB0008847096"],
+            "client_internal": ["imd_43535553", "imd_34534555", "imd_34634673"]
+        })
+
+        scope = "TestScope1"
+        mapping_required = {"name": "instrument_name"}
+        identifier_mapping = {"Figi": "figi", "Isin": "isin", "ClientInternal": "client_internal"}
+
+        # set up initial instruments
+        responses = cocoon.cocoon.load_from_data_frame(
+            api_factory=self.api_factory,
+            scope=scope,
+            data_frame=data_frame,
+            mapping_required=mapping_required,
+            mapping_optional={},
+            file_type="instruments",
+            identifier_mapping=identifier_mapping
+        )
+
+        self.assertGreater(len(responses["instruments"]["success"][0].values), 0)
```

Comment:
Why do you only look at the first response in success? What if your batch size is 1?